.github/workflows/release_nightly.yml

```diff
@@ -406,32 +399,7 @@ jobs:
         working-directory: web
         run: npm run docs
```

Comment:
I believe this should be removed from here, and kept in the place where I made my initial comment about the docs.

Reply:
Removed, thank you.